### DISCUSSION TITLE
Pyinstaller Compiling: Hidden Import Not Found?

### DISCUSSION THREAD

**Discussion by ez4bk** [ultralytics/yolov5#11097] [Q&A]
I'm writing app to utilize a yolov5 model and trying to package it using Pyinstaller. But Pyinstaller seems not to support the ultralystics package. I put it in the hidden import option and the log shows no module found to be ultralytics. Do you guys have any ideas how to compile this yolov5 with pyinstaller? Or is there another way to package my app?
<img width="563" alt="image" src="https://user-images.githubusercontent.com/86708505/222327049-b41a8724-ce3c-475a-afac-af68d52dff42.png">

**Comment by Simzie** [comment]
I just happened across your question and since no-one else has helped you out I thought I would pitch in, although youve most likely worked things out by now. I think you're most likely confusing yolov5 (no pip installer, you run python scripts) for yolov8 (installed as 'ultralytics' not 'ultralystics'). Yolov5 inference is typically performed with torch hub, so you would want pyinstaller to bundle torch, while yolov8 uses the ultralytics package.
